stable/elasticsearch-exporter/README.md

```diff
@@ -67,7 +67,14 @@ Parameter | Description | Default
 `es.ssl.client.key` | Private key for client auth when connecting to Elasticsearch |
 `web.path` | path under which to expose metrics | `/metrics`
 `serviceMonitor.enabled` | If true, a ServiceMonitor CRD is created for a prometheus operator | `false`
-`serviceMonitor.labels` | Labels for prometheus operator | `{}`
+`serviceMonitor.labels` | ServiceMonitor labels for prometheus operator | `{}`
+`prometheusRule.enabled` | If true, a PrometheusRule CRD is created for a prometheus operator | `false`
+`prometheusRule.prometheusNamespace` | Namespace for prometheusRule | `monitoring`
+`prometheusRule.elasticsearchNodes` | Number of all nodes in Elasticsearch cluster (all roles) | `7`
+`prometheusRule.elasticsearchDateNodes` | Number of Data nodes in Elasticsearch cluster | `2`
```

Comment:
Typo! + Could this be inferred from the values?

Reply:
Fixed typo